istio-proxy/istio-proxy.spec

```diff
@@ -2,14 +2,16 @@
 %global with_devel 0
 # Build with debug info rpm
 %global with_debug 0
+#Build stripped binaries
+%global with_stripped_binaries 1
```

Comment:
Couldn't we use with_debug? I think with_debug == 0 implies stripping of binaries. Alternatively, we could move all devel/debug stuff into a dedicated package...